kymatio/scattering1d/filter_bank.py

```diff
@@ -456,8 +202,13 @@ def compute_params_filterbank(sigma_low, Q, r_psi=math.sqrt(0.5), alpha=5.):
     PhD Thesis, 2017
     https://tel.archives-ouvertes.fr/tel-01559667
     """
-    xi_max = compute_xi_max(Q)
-    sigma_max = compute_sigma_psi(xi_max, Q, r=r_psi)
+    xi_max = max(1. / (1. + math.pow(2., 3./Q)), 0.35)
+
+    factor = 1. / math.pow(2, 1. / Q)
```

Comment:
Are there more descriptive names for these variables?

Reply:
At least motivate a bit.

Reply:
addressed in 6c507fd
note that i have changed the formula to 
xi_max = max(1. / (3. - math.pow(2., 1./Q)), 0.39)

because i couldn't make sense of the one i found